VS.Data.Sqlite.IntegrationTests/MappedObjectTests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.VisualStudio.Data.Services;
+using Microsoft.VisualStudio.Data.Services.RelationalObjectModel;
+using Microsoft.VisualStudio.Data.Services.SupportEntities;
+using Microsoft.VisualStudio.Shell;
+using Xunit;
+
+namespace Microsoft.VisualStudio.Data.Sqlite
+{
+    public class MappedObjectTests
+    {
+        [VsFact]
+        public void Can_select_tables()
+        {
+            var connectionFactory = (IVsDataConnectionFactory)ServiceProvider.GlobalProvider.GetService(typeof(IVsDataConnectionFactory));
+            var connection = connectionFactory.CreateConnection(
+                PackageGuids.guidSqliteDataProvider,
+                "Data Source=:memory:",
+                encryptedString: false);
+
+            var command = (IVsDataCommand)connection.GetService(typeof(IVsDataCommand));
+            command.ExecuteWithoutResults(@"
+                CREATE TABLE A (X);
+                CREATE TABLE B (X);
+            ");
+
+            var selector = (IVsDataMappedObjectSelector)connection.GetService(typeof(IVsDataMappedObjectSelector));
+            var tables = selector.SelectMappedObjects<IVsDataTable>();
```

Comment:
@ErikEJ FYI, this API to gets [well-known objects](https://docs.microsoft.com/dotnet/api/microsoft.visualstudio.data.services.relationalobjectmodel) (tables, views, TVFs, sprocs, etc.) from a DDEX provider. The EF6 designer uses it for sprocs and functions, but not tables and views since it's running in-process and can use EF to get those.